Bikes/charts/bikes/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 image:
   repository: azdspublic/bikesharing-bikes
   tag: latest
```

Comment:
>latest [](start = 7, length = 6)

(from offline discussion)

To support this change, we should start specifying specific image tags in these charts.  I've logged #848677 to address this.  Feel free to include in a separate PR if it makes more sense!

Reply:
Thanks for your comment and the interesting offline discussion! I believe it makes sense to fix this bug with the PR, as these changes are clearly related.

---
In reply to: [275914474](https://github.com/ContosoBikeRental/BikeSharingSampleApp/pull/34#discussion_r275914474) [](ancestors = 275914474)